packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.test.tsx

```diff
@@ -42,8 +46,8 @@ describe('ResetPasswordWithRecoveryKeyVerified', () => {
     );
     fireEvent.click(newAccountRecoveryKeyButton);
     expect(logViewEvent).toHaveBeenCalledWith(
-      viewName,
-      `${viewName}.generate-new-key`,
+      `flow.${viewName}`,
```

Comment:
Changed to match event in `amplitude.js`

Reply:
Nice catch!